microsoft/devhome
# Move QuietBackgroundProcesses UI to Windows customization

## Summary of the pull request
Move QuietBackgroundProcesses UI to Windows customization instead of a top-level page.

## References and relevant issues
![image](https://github.com/microsoft/devhome/assets/7259210/479778ff-6104-4f60-94f7-43030e091a4e)

## Detailed description of the pull request / Additional comments
Merge the UI project for QuietBackgroundProcesses into Customization and control the view's visibility based on the experimental features enablement status.

## Validation steps performed
With the feature disabled in Settings -> Experimental features navigate to Windows customization and see that the Quiet mode card is not visible.

Enable the feature and notice that the settings card is visible upon navigating back to Windows customization.

Start a session, navigate away from the page and back and ensure the session remains active.

Close Dev Home and verify that DevHome.QuietBackgroundProcesses.ElevatedServer.exe is still running, launch, navigate back to Windows customization and stop session. Confirm that everything cleans up.

## PR checklist
